pkg/sentry/platform/kvm/machine.go

```diff
@@ -436,7 +436,7 @@ func (m *machine) Get() *vCPU {
 			}
 
 			// The vCPU is not be able to transition to
-			// vCPUGuest|vCPUUser or to vCPUUser because that
+			// vCPUGuest|vCPUWaiter or to vCPUUser because that
```

Comment:
I don't understand this change. Could you explain?

Reply:
> I don't understand this change. Could you explain?

`// vCPUGuest|vCPUUser or to vCPUUser because that`
I read the following code, from my point of view, this comment refers to the following code `c.waitUntilNot(vCPUGuest | vCPUWaiter)` and  `atomic.CompareAndSwapUint32(&c.state, vCPUReady, vCPUUser)`. If my understanding is right, `vCPUGuest|vCPUUser` should change to `vCPUGuest|vCPUWaiter`.

Reply:
> I don't understand this change. Could you explain?

Hi Andrei, if my understanding is wrong, I can drop this commit

Reply:
Maybe you are right. Let me think.

Reply:
vCPUGuest|vCPUUser  is set in bluepillHandler and the machine mutex is unlocked at that moment. I think you are right.